lib/schedule/gtfs/route.ex

```diff
@@ -61,26 +75,25 @@ defmodule Schedule.Gtfs.Route do
     short_name
   end
 
-  @spec bus_route_row?(Csv.row()) :: boolean
-  defp bus_route_row?(row) do
-    # Verify that "route_type" exists on the row, especially to prevent issues while testing
-    if row["route_type"] == nil do
-      raise ArgumentError, message: "route_type is required on route rows"
-    end
+  @spec row_has_route_type?(Csv.row()) :: boolean()
+  # Verify that "route_type" exists on the row, especially to prevent issues while testing
+  defp row_has_route_type?(%{"route_type" => nil}) do
+    raise ArgumentError, message: "route_type is required on route rows"
+  end
 
-    row["route_type"] == "3"
+  defp row_has_route_type?(_route_row) do
+    true
   end
 
-  @spec bus_route_mbta?(Csv.row()) :: boolean
-  defp bus_route_mbta?(row) do
-    # Verify that route number is not one of the private carriers: 710, 712, 713, 714, 716
-    row["route_id"] not in ["710", "712", "713", "714", "716"]
+  @spec bus_route?(t()) :: boolean
+  def bus_route?(route) do
+    route.type == 3
```

Comment:
**nitpick:** we currently classify the SL1/2/3 as route-type 3 (regular bus), but since we do still operate those routes that run at least partially as trolleybuses we should also include `route_type` 11 in case something about our GTFS data ever changes.

Reply:
I'm inclined to keep this as 3 to match the previous row-level [bus filtering logic here](https://github.com/mbta/skate/pull/1846/files#diff-d659be3f7889c470bc13601bacd56a935912ad2857bc9996bdf60a168ec3c3b9L71) in order to keep this PR a pure refactor with no functionality change. Does that sound alright?

Reply:
Oh, that's so strange, I wonder why that didn't break with the 71/73 back when they were actually trolleybuses.